### ISSUE TITLE
a beginner's question: Should the bluetooth version match the linux kernel version?

### ISSUE THREAD

**Issue by HoweverWang** [jeremyb31/bluetooth-6.5#1]
I'm about to update my kernel to 6.6, but I couldn't find any "bluetooth 6.6" on github.
I'm wondering if the bluetooth version means the kernel version? "bluetooth 6.5" means it should be installed in a system of linux kernel 6.5?
And, if you know any place to get "bluetooth 6.6", I'd be appreciated to hear that.
Again, sorry for the stupid question.

**Comment by jeremyb31** [comment]
Are you having problems with an unsupported Bluetooth device?

**Comment by HoweverWang** [comment]
Yeah, it's a '6252M-PUB Wi-Fi6 Module' from FN-LINK (https://www.fn-link.com/6252M-PUB-Wi-Fi6-Module-pd41939580.html). A USB Bluetooth module is integrated on the chip. FN-LINK provided me with an 'rtl8852bu_fw' and an 'rtl8852bu_config' file, and I placed them in the correct file path. However, the Bluetooth functionality is not working properly. It can scan, pair, and connect successfully, but it gets disconnected almost immediately. As a result, I am unable to listen to music or transfer files from my Windows 10 laptop.

Here are the details of the environment:
Thinkpad X1 (with the original Bluetooth turned off)
Ubuntu 22.04.3 LTS (Linux kernel 6.5.0-15-generic, also tried with kernel 6.6.10 but it failed)
BlueZ 5.64

I'd appreciate any advice or instructions.
